packages/babel-core/src/config/files/configuration.js

```diff
@@ -30,7 +30,7 @@ export function findBabelrc(
   filepath: string,
   envName: string,
 ): ConfigFile | null {
-  const dirname = path.dirname(filepath);
+  const dirname = filepath ? path.dirname(filepath) : process.cwd();
```

Comment:
We could also consider changing this function to accept a `dirname` as the argument. Then you can call

Reply:
That got posted before I intended, sorry.

Reply:
One concern I have about exposing this is that we probably want to change the signature of this function more in the future, and it will be easy to miss that we're breaking a public API. I'm wondering if it would be better to  make a new function right in `src/index.js` that calls through to this. For instance we may add flags to tell a the `.babelrc.js` file whether Babel allows a promise to be returned, and we'll probably add another parameter for passing a `root` folder to stop searching at.